From fe6b903880e58a6a754d04dab834cf283e6df234 Mon Sep 17 00:00:00 2001
From: Ankit Agrawal <ankit.agarwal@intel.com>
Date: Thu, 18 May 2023 16:26:33 +0530
Subject: [PATCH] WA-Fixed Build Error in First Stage Init.

Build Error-:
init: InitFatalReboot: signal 6
init:   #00 pc 000000000015260a  /system/bin/init (android::init::InitFatalReboot(int)+250) (BuildId: 5c66c99280015ce9bd65ef10c04da5ad)
init:   #01 pc 00000000000cfdab  /system/bin/init (android::init::InitAborter(char const*)+219) (BuildId: 5c66c99280015ce9bd65ef10c04da5ad)
init:   #02 pc 000000000001a700  /system/lib64/libbase.so (android::base::SetAborter(std::__1::function<void (char const*)>&&)::$_3::__invoke(char const*)+64) (BuildId: 31ed4ad3aa071d4976284f1d6f5682e3)

PLOG message was getting aborted. So commenting for now
to resolve boot issue.

Tracked-On: OAM-106853
Signed-off-by: Ankit Agrawal <ankit.agarwal@intel.com>
---
 init/first_stage_init.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/init/first_stage_init.cpp b/init/first_stage_init.cpp
index 107e99a315..ec82919c2d 100644
--- a/init/first_stage_init.cpp
+++ b/init/first_stage_init.cpp
@@ -113,7 +113,7 @@ static void Copy(const char* src, const char* dst) {
         LOG(INFO) << "hard linking " << src << " to " << dst << " succeeded";
         return;
     }
-    PLOG(FATAL) << "hard linking " << src << " to " << dst << " failed";
+    //PLOG(FATAL) << "hard linking " << src << " to " << dst << " failed";
 }
 
 // Move snapuserd before switching root, so that it is available at the same path
-- 
2.17.1

